apps/docs/pages/guides/resources/migrating-to-supabase/postgres.mdx

```diff
@@ -0,0 +1,95 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  title: 'Migrate from Postgres to Supabase',
+  description: 'Migrate your Postgres database to Supabase.',
+}
+
+This is a guide for migrating your Postgres database to [Supabase](https://supabase.com).
+Supabase is one of the best free Postgres providers. Supabase provide all the backend features developers need to build a product: a Postgres database, authentication, instant APIs, edge functions, realtime subscriptions, and storage. Postgres is the core of Supabase—for example, you can use row-level security and there are more than 40 Postgres extensions available.
```

Comment:
Unsure about the wording around "Supabase is one of the best free Postgres providers ..." within our own docs. Perhaps we can use the same wording as the other migration guides, like "Supabase is a robust and open-source platform ..."?